kmir/src/kmir/kmir.py

```diff
@@ -29,24 +29,27 @@ class KMIRSemantics(KCFGSemantics):
     def is_terminal(self, cterm: CTerm) -> bool:
```

Comment:
How come the logic for `is_terminal` doesn't reference the `terminal_rules` function? Where is the latter used elsewhere in the code, if not here?

Reply:
My understanding might not be correct, but my understanding is that the `terminal_rules` function is used to structure or stop `rpc` calls in some way. However, that may be the case for [evm-semantics](https://github.com/runtimeverification/evm-semantics/blob/6d16eee9a2ca66a141db29a6c8b24dc5fcf61431/kevm-pyk/src/kevm_pyk/kevm.py#L141) because they are referencing it elsewhere, and I am not there yet so it is not used at all.

Reply:
I guess take a look and see if the logic in `is_terminal` that checks for labels that are terminal can be refactored to use the `terminal_rules` function; if it can't then no worries